client/blocks/site/index.jsx

```diff
@@ -101,7 +101,7 @@ const Site = React.createClass( {
 							? translate( 'View site %(domain)s', {
 									args: { domain: site.domain },
 								} )
-							: translate( 'Select site %(domain)s', {
+							: translate( 'Site %(domain)s', {
```

Comment:
:disappointed: 20 existing translations will be lost with this change.
If the string must change, the following option has already been translated **21** times:
`translate( 'View site %(domain)s' )` *ES Score: 7*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).